File: core/src/main/java/com/mcreater/amclcore/api/mihoyo/genshin/cn/GenshinGachaApi.java
package com.mcreater.amclcore.api.mihoyo.genshin.cn;

import com.mcreater.amclcore.api.mihoyo.genshin.AbstractGenshinGachaApi;

public class GenshinGachaApi implements AbstractGenshinGachaApi {
    public String getGenshinGachaUrl() {
        return "sg-public-api.hoyolab.com/event/simulatoros/config";
    }
}

File: core/src/main/java/com/mcreater/amclcore/api/mihoyo/genshin/cn/GenshinGachaApi.kt
package com.mcreater.amclcore.api.mihoyo.genshin.cn

import com.mcreater.amclcore.api.mihoyo.genshin.AbstractGenshinGachaApi

class GenshinGachaApi : AbstractGenshinGachaApi {
    override fun getGenshinGachaUrl(): String {
        return "sg-public-api.hoyolab.com/event/simulatoros/config"
    }
}